src/index.js

```diff
@@ -52,20 +55,24 @@ const rrfConfig = {
 firebase.initializeApp(firebaseConfig);
 const analytics = firebase.analytics();
 firebase.auth();
+firebase.firestore();
```

Comment:
what is this line doing?

Reply:
https://react-redux-firebase.com/docs/getting_started.html

It is probably done by `reduxFirestore` behind the scenes but we are no longer using that.